src/items/class-features/elite_saboteur_(ex).json

```diff
@@ -0,0 +1,120 @@
+{
+  "_id": "lBt6FxBpnN2IZDRx",
+  "name": "Elite Saboteur (Ex)",
+  "type": "feat",
+  "img": "icons/svg/mystery-man.svg",
+  "system": {
+    "type": "",
+    "ability": null,
+    "abilityMods": {
+      "parts": []
+    },
+    "actionTarget": "",
+    "actionType": "",
+    "activation": {
+      "type": "",
+      "condition": "",
+      "cost": 0
+    },
+    "area": {
+      "effect": "",
+      "shape": "",
+      "units": "",
+      "value": 0
+    },
+    "attackBonus": 0,
+    "chatFlavor": "",
+    "critical": {
+      "effect": "",
+      "parts": []
+    },
+    "damage": {
+      "parts": []
+    },
+    "damageNotes": "",
+    "description": {
+      "chat": "",
+      "gmnotes": "",
+      "short": "",
+      "unidentified": "",
+      "value": "<p>&gt;@UUID[Compendium.sfrpg.class-features.On2ZrPJTe8FURsXY]{Operative Exploit}</p>\n<p>You can attempt an @Check[type:engineering] to disable a device on a lock or trap in half the normal time. If this would reduce the duration to less than 1 round, you can disable the device as a standard action.</p>"
```

Comment:
For things like this, you should add a Skill Check in the details tab so it appears on the chat card